R/RcppExports.R

```diff
@@ -0,0 +1,39 @@
+# Generated by using Rcpp::compileAttributes() -> do not edit by hand
+# Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
+
+#' Calculate Faith's PD 
+#' 
+#' This function calculates Faith's phylogenetic diversity for a given assay
+#' and rowTree, using a C++ implementation of the Stacked Faith's Phylogenetic
+#' Diversity (SFPhD) algorithm.
+#' 
+#' @details
+#' This function makes several assumptions about the contents of
+#' \code{assay} and \code{rowTree}, namely that:
+#' \itemize{
+#'  \item \code{assay} and \code{rowTree} are both non-empty.
+#'  \item \code{assay} has row and column names.
+#'  \item \code{rowTree}'s nodes are arranged in cladewise order.
+#' }
+#' These checks should all be handled in the surrounding R code.
+#' 
+#' The values returned by this function are equivalent to the values returned
+#' by \code{picante::pd()} with the parameter \code{include.root=TRUE}.
+#' 
+#' The C++ code was adapted from an implementation by the Unifrac team (see
+#' \url{https://genome.cshlp.org/content/31/11/2131} or
+#' \url{https://github.com/biocore/unifrac}), which is licensed under the BSD
```

Comment:
I suggest to add instead a bibliographic information in the @references section and then cite here with (Lastname, year) instead. That give better recognition for the original authors.

Or even better, set up references with this
https://cran.r-project.org/web/packages/Rdpack/vignettes/Inserting_bibtex_references.pdf